CMakeLists.txt

```
@@ -335,15 +335,20 @@ else()
     enable_testing()
 
     find_package(Python COMPONENTS Interpreter)
-    if(Python_FOUND)
+    find_program(LIT_EXECUTABLE NAMES lit lit.py)
```

```suggestion
    find_program(LIT_EXECUTABLE NAMES lit-script.py lit.py lit)
```
If you change it to this (and do `pip install lit` in the build definition) it _might_ work on windows

---

The tests are working on Windows already :). Adding another spelling is fine.

---

It appears this PR is already successfully running the `lit` tests on Windows: https://github.com/google/bloaty/pull/260/checks?check_run_id=3243382953

---

hm, I could have sworn that they were red, guess I looked at old or wrong builds

---

On my dev machine this does not work:
```
5>Running bloaty tests...
5>SyntaxError: Non-UTF-8 code starting with '\x90' in file C:\Program Files\Python39\Scripts\lit.exe on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
5>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(238,5): error MSB8066: Custom build for 'E:\Projects\bloaty\build\CMakeFiles\16d0e534f5107e26a948780ce7ba2662\check-bloaty.rule' exited with code 1.
5>Done building project "check-bloaty.vcxproj" -- FAILED.
```

---

For my dev machine to pick this up, `lit-script.py` needs to come before the `lit` entry.